master/internal/webhooks/postgres_webhook.go

```diff
@@ -286,19 +286,30 @@ func GetWebhook(ctx context.Context, webhookID int) (*Webhook, error) {
 	return &webhook, nil
 }
 
-// GetWebhooks returns all Webhooks from the DB.
-func GetWebhooks(ctx context.Context) (Webhooks, error) {
+// getWebhooks returns Webhooks from the DB whose scopes are in workspaceIDs.
```

Comment:
Sorry, I know I commented on this comment yesterday, but I just wanted to add a little bit more to it for clarity!
Can we maybe say 
`// getWebhooks returns all global webhooks from the DB and all webhooks whose scopes are in workspaceIDs.`
I was confused before as to why users with no Viewer (or any) permissions were able to see global webhooks in the e2e tests, and this I think helps to clarify that!